official/utils/flags/_performance.py

```diff
@@ -276,4 +280,11 @@ def _check_fp16_implementation(flags_dict):
         name="enable_xla", default=False,
         help="Whether to enable XLA auto jit compilation")
 
+  if force_run_distributed:
+    flags.DEFINE_boolean(
+        name="force_run_distributed", default=False,
```

Comment:
Everything looks great except that the name of the flag looks a little confusing to me at first glance (maybe also to others without enough context). Maybe rename it `run_default_strategy` or `force_dist_strat_code_path`?

Reply:
+1 agree. I think the original arg in model.compile is also confusing but they are planning to rename it. we should give this new flag a better name to start with. They are considering something like "run_v2". So maybe this can be something like "force_v2_in_keras_compile" ?

Reply:
I am going to go with Pryia's because it is cryptic and if it stays in I don't want people using it.  I would rather it be confusing.